newIDE/app/src/ResourcesEditor/ResourcePropertiesEditor/index.js

```diff
@@ -38,10 +46,11 @@ export default class ResourcePropertiesEditor extends React.Component<
     },
     {
       name: 'File',
-      valueType: 'string',
+      valueType: 'string-with-button',
```

Comment:
You can keep `string`, as button is already supported by all fields with the new for a new field type :)